src/cmt_decode_opentelemetry.c

```diff
@@ -200,17 +199,6 @@ static int clone_kvlist_entry(struct cmt_kvlist *target,
     return CMT_DECODE_OPENTELEMETRY_SUCCESS;
 }
 
-static void destroy_label(struct cmt_map_label *instance)
```

Comment:
This was there for parity with the creator but in the end wasn't used, maybe we should have these somewhere else since it's not the first time I used them.